alphadia/search_step.py

```diff
@@ -136,7 +136,7 @@ def _load_default_config():
         default_config_path = os.path.join(
             os.path.dirname(__file__), "constants", "default.yaml"
         )
-        logger.info(f"loading config from {default_config_path}")
+        logger.info(f"loading default config from {default_config_path}")
```

Comment:
I'm wondering, this is the default config. should we actually list the path at all as it's ot supposed to be changed (This could make it look like it).